source/vstd/slice.rs

```diff
@@ -39,6 +50,27 @@ pub exec fn slice_index_get<T>(slice: &[T], i: usize) -> (out: &T)
     &slice[i]
 }
 
+////// Len (with autospec)
+pub open spec fn spec_slice_len<T>(slice: &[T]) -> usize;
+
+// This axiom is slightly better than defining spec_slice_len to just be `slice@.len() as usize`
+// (the axiom also shows that slice@.len() is in-bounds for usize)
+#[verifier(external_body)]
+pub broadcast proof fn axiom_spec_len<T>(slice: &[T])
```

Comment:
This will now need to be added to the `vstd_default` broadcast group to be visible to users.